data/plugins/drawwhitespace.lua

```diff
@@ -184,6 +192,13 @@ local function get_option(substitution, option)
   return substitution[option]
 end
 
+local function sort_positions(line1, col1, line2, col2)
+  if line1 > line2 or line1 == line2 and col1 > col2 then
+    return line2, col2, line1, col1, true
+  end
+  return line1, col1, line2, col2, false
+end
+
```

Comment:
No need for this, `Doc:get_selections` can sort by passing `true` as its first parameter.